ljbuild.make

```diff
@@ -0,0 +1,96 @@
+.SUFFIXES:
+
+ifndef name
+$(error build name is not set)
+endif
+
+BUILD_DIR:= builds/$(name)
+BUILD_FULLDIR:= $(shell cd builds && pwd)/$(name)
+BUILD_DIRS= $(BUILD_DIR) $(BUILD_DIR)/include $(BUILD_DIR)/jit $(BUILD_DIR)/jitlog
+REPO:= $(BUILD_FULLDIR)/src
+SRC:= $(BUILD_DIR)/src/src
+
+# We need a branch name when first checking out the repo
+ifndef branch
+  $(if $(shell if [ -f $(REPO)/.git ]; then echo ok; fi), , $(error No 'branch' variable set))
+endif
+
+# Make sure all output build directories are created before we run anything
+$(info $(shell mkdir -p $(BUILD_DIRS)))
```

Comment:
What does `info` do?

Reply:
It it just prints text to the console idk why i left it wrapped in that. I know that the code was a loop before i simplified it at least. It should be fixed now